autoextract/aio/retry.py

```diff
@@ -155,4 +155,4 @@ def build(self):
         )
 
 
-autoextract_retry = RetryFactory().build()
+autoextract_retrying: AsyncRetrying = RetryFactory().build()
```

Comment:
Probably the last question/nitpick :) I *think* that so far there was nothing about async in this retry.py module. The idea was to reuse it in a sync part as well; it is a nice thing about tenacity that it allows to decouple the strategy of retrying from the way it is applied.

Reply:
If I'm not mistaken, autoextract_retrying is a wrapper, not a AsyncRetrying instance. I was wrong about Retrying / AsyncRetrying thing, we're not using them directly.

Reply:
This is the place where tenacity decides which `Retrying` to use: https://github.com/jd/tenacity/blob/master/tenacity/__init__.py#L120-L126 . The result is a wrapper. 

Before the last commit, `autoextract_retry` was a wrapper created with `retry`, and then compatible with sync/async. But the last commit breaks the compatibility: explicit `AsyncRetrying` was introduced. I don't see an elegant way of keeping both things (explicit `Retrying` but with autodetection of sync/async). 

@kmike So then should I revert the commit and leave it as it was before?

Reply:
I see, thanks for the explanation. I think that's ffine to keep it as-is. If we decide to reuse retries for the sync part, we can have build_sync / build_async methods.